src/form-group.scss

```diff
@@ -17,3 +17,12 @@ $block: #{$fd-namespace}-form-group;
     margin-bottom: 0;
   }
 }
+
+.#{$block}--inline {
+  display: flex;
+  justify-content: flex-start;
+
+  > :not(:last-child) {
+    margin-right: fd-space(4);
+  }
```

Comment:
This is scss, you can use nesting to specify that an element is a child of another element or `&`